### PR TITLE
Proofreading SAP2

### PR DESCRIPTION
**Pull request by Karuna-Mendix**
https://mendix.atlassian.net/jira/software/c/projects/TW/issues/TW-2285?filter=myopenissues

### CODE REVIEW

File: content/en/docs/developerportal/deploy/sap-btp/sap-autoscaler.md [hunk 1]
```diff
@@ -12,9 +12,9 @@ aliases:
 
 ## 1 Introduction
 
-The **Application Autoscaler** service for [SAP Business Technology Platform (SAP BTP)](https://docs.mendix.com/developerportal/deploy/sap-cloud-platform/) enables applications to automatically increase or decrease the number of instances of your app. This service can be based on a schedule, or based on the value of a metric.
+The application autoscaler service for [SAP Business Technology Platform (SAP BTP)](https://docs.mendix.com/developerportal/deploy/sap-cloud-platform/) enables applications to automatically increase or decrease the number of instances of your app. This service can be based on a schedule, or based on the value of a metric.
```
Comment (Karuna-Mendix) [inline content/en/docs/developerportal/deploy/sap-btp/sap-autoscaler.md]: Verified with developer that 'application autoscaler' is not an UI text.

Comment (MarkvanMents) [reply content/en/docs/developerportal/deploy/sap-btp/sap-autoscaler.md]: Great to remove the bold.
I'm not sure that it is the _service_ which is based on the schedule - it is the automatic change of resources that is based on the schedule.
Perhaps
```suggestion
The application autoscaler service for [SAP Business Technology Platform (SAP BTP)](https://docs.mendix.com/developerportal/deploy/sap-cloud-platform/) enables applications to automatically increase or decrease the number of instances of your app. These changes can be based on a schedule, or based on the value of a metric.
```

File: content/en/docs/developerportal/deploy/sap-btp/sap-autoscaler.md [hunk 2]
```diff
@@ -12,102 +12,102 @@ aliases:
...
+For more information on the application autoscaler, see [Application Autoscaler Service](https://help.sap.com/docs/CSI/b651ed9a533d41339f05b12549d8d706/c5d349a5ea354971839b7922d13c04e4.html?q=application%20autoscaler%20service) document on the SAP Help Portal.
 
 ## 2 Binding the Application Autoscaler{#binding}
 
-The autoscaler service can be found on the [services tab](/developerportal/deploy/sap-cloud-platform/#binding-services) of the environment details page of your app's environment.
+The **autoscaler** service can be found on the [Services Tab](/developerportal/deploy/sap-cloud-platform/#binding-services) of the environment details page of your app's environment.
 
-Selecting the autoscaler service will expand the entry to allow you to enter more details about the service you wish to bind, namely the **Plan** and the **File** which contains the configuration.
+Selecting the **autoscaler** service, expands the entry to allow you to enter more details about the service you wish to bind, namely the **Plan** and the **File** which contains the configuration.
```
Comment (MarkvanMents) [inline content/en/docs/developerportal/deploy/sap-btp/sap-autoscaler.md]: Don't think we need a comma before "expands", it is not an introductory phrase.

File: content/en/docs/developerportal/deploy/sap-btp/sap-autoscaler.md [hunk 3]
```diff
@@ -12,102 +12,102 @@ aliases:
...
 
 {{< figure src="/attachments/developerportal/deploy/sap-autoscaler/autoscaler-service.png" alt="Autoscaler service on the environment details page" >}}
 
-You will need to specify the following:
+You need to specify the following:
 
 * **Plan** – the plan you wish to use for the autoscaler, either *Lite* or *Standard*
-* **File** – the file which contains the JSON to configure the autoscaler — more information, including the contents of the configuration file can be found in [Application Autoscaler Service](https://help.sap.com/viewer/7472b7d13d5d4862b2b06a730a2df086/Cloud/en-US/4ad999a0be664160a08514ba4ce6430c.html) on the *SAP Help Portal*
+* **File** – the file which contains the JSON to configure the autoscaler — more information, including the contents of the configuration file can be found in [Application Autoscaler Service](https://help.sap.com/docs/CSI/b651ed9a533d41339f05b12549d8d706/c5d349a5ea354971839b7922d13c04e4.html?q=application%20autoscaler%20service) on the SAP Help Portal
```
Comment (MarkvanMents) [inline content/en/docs/developerportal/deploy/sap-btp/sap-autoscaler.md]: This new link does not contain any information about the JSON file. Can you check with the development team if this is a better suggestion?
https://help.sap.com/docs/application-autoscaler/application-autoscaler/defining-scaling-policy

File: content/en/docs/developerportal/deploy/sap-btp/sap-autoscaler.md [hunk 4]
```diff
@@ -12,102 +12,102 @@ aliases:
...
 * Dynamic Scaling – the app is scaled depending on the value of a metric
 * Custom – a combination of both scheduled and dynamic scaling
 
 The metrics you can use to trigger dynamic scaling are as follows:
 
-* Memory consumed: monitors memory consumption in megabytes (memoryconsumed) or in percentage (memoryutil)
-* CPU Utilization in percentage
-* Response time: monitors response time in milliseconds
-* Throughput: monitors requests per seconds (RPS)
+* Memory consumed - monitors memory consumption in megabytes (memoryconsumed) or in percentage (memoryutil)
```
Comment (MarkvanMents) [inline content/en/docs/developerportal/deploy/sap-btp/sap-autoscaler.md]: These hyphens should be n-dashes `–` not hyphens `-`
```suggestion
* Memory consumed – monitors memory consumption in megabytes (memoryconsumed) or in percentage (memoryutil)
```

File: content/en/docs/developerportal/deploy/sap-btp/sap-autoscaler.md [hunk 5]
```diff
@@ -12,102 +12,102 @@ aliases:
...
-You will first need to decide the sort of scaling you want to apply.
+You first need to decide the sort of scaling you want to apply.
 
-For **Scaling** select one of:
+For **Scaling**, select one of the following:
 
-* Schedule-based scaling – scaling based on a schedule, see [Scheduled Scaling](#schedule), below
-* Dynamic scaling – scaling based on the values of metrics, see [Dynamic Scaling](#dynamic), below
-* Custom – you will need to set up both [Dynamic Scaling](#dynamic) and [Scheduled Scaling](#schedule) as described in the sections below
+* Schedule-based scaling – scaling based on a schedule, see [Scheduled Scaling](#schedule) section below
```
Comment (MarkvanMents) [inline content/en/docs/developerportal/deploy/sap-btp/sap-autoscaler.md]: You need a 'the' here. You can see "Scheduled Scaling", but you see "the Scheduled Scaling section"
```suggestion
* Schedule-based scaling – scaling based on a schedule, see the [Scheduled Scaling](#schedule) section below
```

File: content/en/docs/developerportal/deploy/sap-btp/sap-autoscaler.md [hunk 6]
```diff
@@ -12,102 +12,102 @@ aliases:
...
+* Dynamic scaling – scaling based on the values of metrics, see [Dynamic Scaling](#dynamic)section below
```
Comment (MarkvanMents) [inline content/en/docs/developerportal/deploy/sap-btp/sap-autoscaler.md]:
```suggestion
* Dynamic scaling – scaling based on the values of metrics, see [Dynamic Scaling](#dynamic) section below
```

File: content/en/docs/developerportal/deploy/sap-btp/sap-autoscaler.md [hunk 7]
```diff
@@ -12,102 +12,102 @@ aliases:
...
+You then need to set up the requested type of scaling. The sections below show a summary of the required information. For more information, see the [Application Autoscaler Service](https://help.sap.com/docs/CSI/b651ed9a533d41339f05b12549d8d706/c5d349a5ea354971839b7922d13c04e4.html?q=application%20autoscaler%20service) documentation on the SAP Help Portal.
 
 When you have added all the rules, you can do one of the following:
 
 * Click **Upload Configuration To Service** to upload your autoscaler configuration directly to the autoscaler service.
-* Click **Download Configuration File** to create a file containing the correct JSON for your autoscaler configuration. You can then upload this file as the configuration of your autoscaler service.
+* Click **Download Configuration File** to create a file containing the correct JSON for your autoscaler configuration; you can then upload this file as a configuration of your autoscaler service.
```
Comment (MarkvanMents) [inline content/en/docs/developerportal/deploy/sap-btp/sap-autoscaler.md]: I think this is still **the** configuration. Although you can create multiple configurations, I think you can only upload one configuration to the service.
```suggestion
* Click **Download Configuration File** to create a file containing the correct JSON for your autoscaler configuration; you can then upload this file as the configuration of your autoscaler service.
```

File: content/en/docs/developerportal/deploy/sap-btp/sap-autoscaler.md [hunk 8]
```diff
@@ -12,102 +12,102 @@ aliases:
...
 
 ### 3.1 Scheduled Scaling{#schedule}
 
 {{% alert color="warning" %}}
-Do not attempt to enter overlapping schedules as this will cause the scaling to fail.
+Do not attempt to enter overlapping schedules, as it causes the scaling to fail.
```
Comment (MarkvanMents) [inline content/en/docs/developerportal/deploy/sap-btp/sap-autoscaler.md]: I don't think this comma is needed.

**Review by MarkvanMents — CHANGES_REQUESTED** [review]
Hi Karuna (@Karuna-Mendix),
Lots of good changes here.
I've made a couple of comments, can you look at them and let me know if you have any questions about what I've said.

File: content/en/docs/developerportal/deploy/sap-btp/sap-autoscaler.md (hunk 1 above)
Comment (Karuna-Mendix) [reply content/en/docs/developerportal/deploy/sap-btp/sap-autoscaler.md]: I agree. Corrected in https://github.com/mendix/docs/pull/6673/commits/be785809847f6b0b1d0a54e88bccaceb697d2a72

File: content/en/docs/developerportal/deploy/sap-btp/sap-autoscaler.md (hunk 2 above)
Comment (Karuna-Mendix) [reply content/en/docs/developerportal/deploy/sap-btp/sap-autoscaler.md]: done https://github.com/mendix/docs/pull/6673/commits/be785809847f6b0b1d0a54e88bccaceb697d2a72

File: content/en/docs/developerportal/deploy/sap-btp/sap-autoscaler.md (hunk 4 above)
Comment (Karuna-Mendix) [reply content/en/docs/developerportal/deploy/sap-btp/sap-autoscaler.md]: done https://github.com/mendix/docs/pull/6673/commits/be785809847f6b0b1d0a54e88bccaceb697d2a72

File: content/en/docs/developerportal/deploy/sap-btp/sap-autoscaler.md (hunk 6 above)
Comment (Karuna-Mendix) [reply content/en/docs/developerportal/deploy/sap-btp/sap-autoscaler.md]: done https://github.com/mendix/docs/pull/6673/commits/be785809847f6b0b1d0a54e88bccaceb697d2a72

File: content/en/docs/developerportal/deploy/sap-btp/sap-autoscaler.md (hunk 7 above)
Comment (Karuna-Mendix) [reply content/en/docs/developerportal/deploy/sap-btp/sap-autoscaler.md]: done https://github.com/mendix/docs/pull/6673/commits/be785809847f6b0b1d0a54e88bccaceb697d2a72

File: content/en/docs/developerportal/deploy/sap-btp/sap-autoscaler.md (hunk 3 above)
Comment (Karuna-Mendix) [reply content/en/docs/developerportal/deploy/sap-btp/sap-autoscaler.md]: Verified this link with developer and corrected. https://github.com/mendix/docs/pull/6673/commits/61678ca354771c2dc2d3f72944b74c1494af416b

**Review by MarkvanMents — APPROVED** [review]
@Karuna-Mendix 
This looks great. Approving.
Feel free to merge whenever you want.

**Pull request merged**